pyproject.toml

```diff
@@ -23,3 +23,12 @@ enableTypeIgnoreComments = true
 reportMissingImports = "none"
 # Some modules are missing type stubs, which is an issue when running pyright locally
 reportMissingTypeStubs = false
+
+[tool.pytest.ini_options]
+
+# Throw errors on warnings, except for some specific warnings.
+filterwarnings = [
+    "error",
```

Comment:
Why are we filtering errors?

Reply:
The "error" in `filterwarnings` means to turn warnings into errors ([source](https://docs.python.org/3/library/warnings.html#warning-filter)). I have edited the comment to clarify this.